content/zh-cn/docs/_index.md

```diff
@@ -8,41 +8,36 @@ menu:
 lastmod: 2019-05-15
 ---
 
-# 概括
+# 概述
 
-* [入门]({{< ref "/getting-started.md" >}})
-* [Let's Encrypt工作方法]({{< ref "/how-it-works.md" >}})
-* [常问问题 (FAQ)]({{< ref "/docs/faq.md" >}})
-* [Glossary]({{< ref "/docs/glossary.md" >}})
+* [快速入门]({{< ref "/getting-started.md" >}})
+* [Let's Encrypt运作方式]({{< ref "/how-it-works.md" >}})
+* [常见问题（FAQ）]({{< ref "/docs/faq.md" >}})
+* [术语表]({{< ref "/docs/glossary.md" >}})
 
-# 用户（订户）信息
+# 用户信息
 
 * [ACME客户端]({{< ref "/docs/client-options.md" >}})
 * [速率限制]({{< ref "/docs/rate-limits.md" >}})
 * [过期提醒邮件]({{< ref "/docs/expiration-emails.md" >}})
 
-# 高级用户（订户）信息
+# 高级用户信息
 
 * [测试环境]({{< ref "/docs/staging-environment.md" >}})
 * [证书兼容性]({{< ref "/docs/cert-compat.md" >}})
-* [证书信任链 (根和中间证书)]({{< ref "/certificates.md" >}})
-* [即将推出功能]({{< ref "/upcoming-features.md" >}})
-
+* [证书信任链（根和中间证书）]({{< ref "/certificates.md" >}})
+* [即将推出的功能]({{< ref "/upcoming-features.md" >}})
 * [吊销证书]({{< ref "/docs/revoking.md" >}})
 * [证书颁发机构授权（CAA）]({{< ref "/docs/caa.md" >}})
-
 * [localhost证书]({{< ref "/docs/certificates-for-localhost.md" >}})
-* [Best Practice - Keep Port 80 Open]({{< ref "/docs/allow-port-80.md" >}})
-
-* [Challenge Types]({{< ref "/docs/challenge-types.md" >}})
-* [Certificate Transparency (CT) Logs]({{< ref "/docs/ct-logs.html" >}})
+* [最佳实践——保持开放80端口]({{< ref "/docs/allow-port-80.md" >}})
+* [验证方式]({{< ref "/docs/challenge-types.md" >}})
+* [证书透明度（CT）日志]({{< ref "/docs/ct-logs.html" >}})
 
 # 客户端开发者信息
 
 * [客户端和大型提供商集成指南]({{< ref "/docs/integration-guide.md" >}})
-
 * [ACME协议更新]({{< ref "/docs/acme-protocol-updates.md" >}})
-
-* [当前ACME草案变化](https://github.com/letsencrypt/boulder/blob/master/docs/acme-divergences.md)
+* [与当前ACME草案的不同](https://github.com/letsencrypt/boulder/blob/master/docs/acme-divergences.md)
```

Comment:
This is just purely weird... However, I've got no better suggestions.

Reply:
Yeah, it's weird. I just thought the original one didn't express the meaning of the English version precisely and can't find a better way to translate it.